public/403.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>The change you wanted was rejected (422)</title>
```

Comment:
Sorry @Alwahsh,
That was my mistake. I copied the content from 422.html. Updated the code. Thank you for your review.